### PR TITLE
Bug 1987137: Explain profile annotation requirements

### PR DESCRIPTION
**Pull request by jottofar**

### CODE REVIEW

File: dev-guide/cluster-version-operator/dev/object-deletion.md [hunk 1]
```diff
@@ -114,3 +115,4 @@ This will be accomplished through the existing CVO precondition mechanism which
 
 [svcat-apiserver-4.4-manifests]: https://github.com/openshift/cluster-svcat-apiserver-operator/tree/aa7927fbfe8bf165c5b84167b7c3f5d9cb394e14/manifests
 [deprecated-marketplace-apis]: https://docs.openshift.com/container-platform/4.4/release_notes/ocp-4-4-release-notes.html#ocp-4-4-marketplace-apis-deprecated
+[cluster-profiles]: https://github.com/openshift/enhancements/blob/ab877949c3be862f342b8e97bf6aad9b93ce15d4/dev-guide/cluster-version-operator/dev/operators.md#specifying-a-cluster-profile-in-your-manifest
```
[the remark this answers is not in the document]
Comment (jottofar) [reply dev-guide/cluster-version-operator/dev/object-deletion.md]: done

File: dev-guide/cluster-version-operator/dev/operators.md [hunk 2]
```diff
@@ -99,12 +99,22 @@ When your manifests are added to the release image, they’ll be given a prefix
...
+
 ### What if I only want the CVO to create my resource, but never update it?
 
 This is only applicable to cases where the contents of a resource are not managed, but the presence is required for
 usability.  Today the only known use-case is config.openshift.io, so that `oc edit foo.config.openshift.io` "just works".
 
-To do this, you can set .metadata.annotations["release.openshift.io/create-only"]="true".
+To do this, you can set the following metadata annotation:
+
+```text
```
[the remark this answers is not in the document]
Comment (jottofar) [reply dev-guide/cluster-version-operator/dev/operators.md]: fixed

File: dev-guide/cluster-version-operator/dev/operators.md [hunk 3]
```diff
@@ -165,3 +175,5 @@ Bundle up your manifests and the metadata file as a docker image and push them t
 Later on, when someone wants to mirror a particular release, there will be tooling that can take the list of all images used by operators and mirror them to a new repo.
 
 This pattern tries to balance between having the manifests in your source repo be able to deploy your latest upstream code *and* allowing us to get a full listing of all images used by various operators.
+
+[cluster-profiles]: https://github.com/openshift/enhancements/blob/master/enhancements/update/cluster-profiles.md
```
[the remark this answers is not in the document]
Comment (jottofar) [reply dev-guide/cluster-version-operator/dev/operators.md:179]: done

Comment (wking) [inline dev-guide/cluster-version-operator/dev/operators.md:179]: no need for this definition, now that you have a local, inline link above.

Comment (jottofar) [reply dev-guide/cluster-version-operator/dev/operators.md]: fixed

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**